tests/utils.test.ts

```diff
@@ -0,0 +1,50 @@
+import { validateIdentityName } from '../src/utils'
+
+describe(validateIdentityName.name, () => {
+  it('returns error state when string less than 8 characters', () => {
+    const result = validateIdentityName('john')
+    expect(result).toEqual('error_minimum_length')
```

Comment:
Nit: Would be cleaner to use the enum here, so we don't have to update tests if the string changes.